doc/sphinx/addendum/type-classes.rst

```diff
@@ -274,6 +279,10 @@ This declares singleton classes for reflexive and transitive relations,
 (see the :ref:`singleton class <singleton-class>` variant for an
 explanation). These may be used as parts of other classes:
 
+.. coqtop:: none
+
+   Set Warnings "future-coercion-class-field".
+
 .. coqtop:: all
 
    Class PreOrder (A : Type) (R : relation A) :=
```

Comment:
This doesn't render well.  Can you tweak how the message is generated so there's a line break?  Also, IIRC I haven't seen the `^^^^^^^^^^^^` for other warnings, is there a way to avoid that?  After addressing that, can you change the example so it doesn't get the warning message?

![image](https://user-images.githubusercontent.com/1253341/158869751-52b45d97-01d7-48e7-bf40-383213f9e31f.png)

Reply:
Let's just silence the warning in the example (like done elsewhere in the refman). The warning is not the goal of this example anyway.